ci-operator/config/openshift/security-profiles-operator/openshift-security-profiles-operator-main.yaml

```diff
@@ -0,0 +1,68 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+canonical_go_repository: sigs.k8s.io/security-profiles-operator
+images:
+- dockerfile_path: Dockerfile.ubi
+  to: security-profiles-operator
+promotion:
```

Comment:
As discussed in the other thread, I'm not sure you need to do mirroring at all. If you don't have a specific reason to do it, like needing the image to be available in other jobs, you could remove these promotion instructions and skip it.

Reply:
Great, I've removed the promotion jobs, which ended up removing the whole `postsubmit` job (fine by me, I only need the CI) and finally the image mapping was removed.
Thank you for your help!